shit.py

```diff
@@ -0,0 +1,2 @@
+if __name__ == '__main__':
+    print('shit')
```

Comment:
LGTM after you have added the shit emoji

```suggestion
    print('shit 💩')
```